audit/projects/kubernetes-public/iam.json

```diff
@@ -26,6 +27,12 @@
       ],
       "role": "roles/bigquery.user"
     },
+    {
+      "members": [
+        "serviceAccount:service-127754664067@gcp-sa-bigquerydatatransfer.iam.gserviceaccount.com"
+      ],
+      "role": "roles/bigquerydatatransfer.serviceAgent"
+    },
```

Comment:
I did a manual run of the transfer job. https://github.com/kubernetes/k8s.io/pull/2737